### ISSUE TITLE
pre-main-kyma-gke-external-registry job is failing

### ISSUE THREAD

**Issue by Halamix2** [kyma-project/kyma#12562]
**Description**

The `pre-main-kyma-gke-external-registry` job is failing

**Steps to reproduce**

Use this PR: https://github.com/kyma-project/kyma/pull/12561

**Troubleshooting**

```bash
kubectl -n test-serverless-gitops-11h-49m-3782159331395911226 logs gitfunc-build-ctv4n-hnwrz
error checking push permissions -- make sure you entered the correct tag name, and that you are authenticated correctly, and try again: checking push permission for "eu.gcr.io/functions/test-serverless-gitops-11h-49m-3782159331395911226-gitfunc:099fdd8035923b903a5590594da164ce5c17704f1db3295e94eb91a320e8e15e": creating push check transport for eu.gcr.io failed: GET https://eu.gcr.io/v2/token?scope=repository%3Afunctions%2Ftest-serverless-gitops-11h-49m-3782159331395911226-gitfunc%3Apush%2Cpull&service=eu.gcr.io: UNKNOWN: Project 'project:functions' not found or deleted.
```

**Comment by adamwalach** [comment]
Last green execution: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/12166/pre-main-kyma-gke-external-registry/1440949260131504128

Please note that recent greens are false positives because job was reporting success even when tests failed.
This is the fix for the job: https://github.com/kyma-project/test-infra/pull/4386

**Comment by Halamix2** [comment]
This job was fixed in kyma-project/test-infra#4408
